packages/p2p-v2/src/components/Modals/OrderDetailsComplainModal/OrderDetailsComplainModalRadioGroup/OrderDetailsComplainModalRadioGroup.scss

```diff
@@ -0,0 +1,19 @@
+.p2p-v2-order-details-complain-modal-radio-group {
+    &:not(.p2p-v2-radio-group__item) {
```

Comment:
the design looked a little different from the screenshots like on desktop, the padding for the modal should be a bit more like in the design, and the Cancel button border width should be smaller on desktop and responsive

**From Design:**
![Screenshot 2024-04-16 at 2 20 35 PM](https://github.com/binary-com/deriv-app/assets/103412909/764def33-3ec7-46af-a182-529c3d4aa92d)

**From your screenshots:**
![Screenshot 2024-04-16 at 2 26 25 PM](https://github.com/binary-com/deriv-app/assets/103412909/953f4ad8-0ce8-42ae-a2e3-b2607826e249)
![Screenshot 2024-04-16 at 2 26 23 PM](https://github.com/binary-com/deriv-app/assets/103412909/50d72663-6530-4959-96fe-1ad67282159d)